tests/test_migrator.py

```diff
@@ -1,23 +1,14 @@
 import metastore.backend as metastore
-from mock import patch
-
 from migrate_metadata import migrator
-from migrate_metadata.ckan_api import CkanAPIClient
 
 
-@patch('migrate_metadata.ckan_api.CkanAPIClient.get_all_datasets')
-def test_migrate_all_datasets(mock_get_all_datasets):
-    mock_reutrn_value = {'result': [{0: {
-                    "name": "test_pkg_0",
-                    "author": "test_user"}},
-                    {1: {
-                    "name": "test_pkg_1",
-                    "author": "test_user"}}
-                    ]}
-    mock_get_all_datasets.return_value = mock_reutrn_value
-    ckan_client = CkanAPIClient("http://test", "xyz-123")
+def test_migrate_datasets_migrates_only_of_type_dataset():
+    datasets = [
```

Comment:
When testing it's a good idea to make the test data naive, but not *too* naive. For example, this test would pass if the code only migrates the 1st item or just always returns 1 or whatever - basically I'm saying "1" is too specific. How about we add a 3rd item here of `dataset` type and check that the return value is 2? This increases the chances of this test finding a bug some day significantly. Or maybe just add a couple more similar tests (with `parametrize`).